src/App.js

```diff
@@ -1,23 +1,33 @@
-import logo from './logo.svg';
+import React, {useLayoutEffect, useState} from 'react';
 import './App.css';
+import {BrowserRouter as Router, Route, Switch} from "react-router-dom";
+import HomePage from './pages/HomePage';
+import LoginPage from './pages/LoginPage';
+import NavBar from './components/NavBar';
 
 function App() {
+  const [background, setBackground] = useState(false);
+  const showBackground = () => setBackground(!background);
```

Comment:
delete unused variables

Reply:
I deleted line 10. I still need line 9, so I keep it.
I also fixed the NavBar pops out issue.